core/src/main/c/share/txn_board.cpp

```diff
@@ -25,127 +25,127 @@
 #include <cstdint>
```

Comment:
There could be races in this scoreboard implementation. At least, I was able to observe releases on transactions that are below the current minimum:
https://github.com/puzpuzpuz/questdb/commit/4f9e0326fbd28c26612fe6055b33fbeda956602d

Could you take a look?

Reply:
As @ideoma pointed out, loads used in `txn_release`'s assertion were incorrectly ordered. So, this one is a false positive failure.